src/effects/native/loudnesscontoureffect.cpp

```diff
@@ -0,0 +1,159 @@
+#include <effects/native/loudnesscontoureffect.h>
+#include "util/math.h"
+
+namespace {
+
+// The defaults are tweaked to match the TEA6320 IC
+static const int kStartupSamplerate = 44100;
+static const double kLoPeakFreq = 20.0;
+static const double kHiShelveFreq = 3000.0;
+static const double kMaxLoGain = 30.0;
+static const double kHiShelveGainDiv = 3.0;
+static const double kLoPleakQ = 0.2;
+static const double kHiShelveQ = 0.7;
+
+} // anonymous namesspace
+
+
+// static
+QString LoudnessContourEffect::getId() {
+    return "org.mixxx.effects.loudnesscontour";
+}
+
+// static
+EffectManifest LoudnessContourEffect::getManifest() {
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("Loudness Contour"));
+    manifest.setShortName(QObject::tr("Loudness"));
+    manifest.setAuthor("The Mixxx Team");
+    manifest.setVersion("1.0");
+    manifest.setDescription(QObject::tr(
+        "A filter to compensate the loudness contour at lower volume"));
+    manifest.setEffectRampsFromDry(true);
+    manifest.setIsMixingEQ(true);
+
+    EffectManifestParameter* low = manifest.addParameter();
```

Comment:
My old Yamaha Amp has such a loudness knob and I miss it on my new 5.1 one where I cannot even disable loudness.  

The use case here is that the loudness level depends on the systems volume, which we cannot control, because Mixxx has no access to the Amp volume.  So the user has the option to pick a suitable value for the current loudness.